Sources/Commands/APIDigester.swift

```diff
@@ -187,6 +206,21 @@ public struct SwiftAPIDigester {
     }
 }
 
+extension SwiftAPIDigester {
+    /// The result of comparing a module's API to a provided baseline.
+    public struct ComparisonResult {
+        /// Breaking changes made to the API since the baseline was generated.
+        var apiBreakingChanges: [SerializedDiagnostics.Diagnostic]
+        /// Other diagnostics emitted while comparing the current API to the baseline.
+        var otherDiagnostics: [SerializedDiagnostics.Diagnostic]
+
+        /// `true` if the comparison succeeded and no breaking changes were found, otherwise `false`.
+        var isSuccessful: Bool {
```

Comment:
Maybe rename 'hasNoAPIBreakingChanges' (or 'hasAPIBreakingChanges' that returns the opposite value)? ComparisonResult.isSuccessful doesn't necessarily convey what's successful.

Reply:
sounds good to me, done!